at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -337,13 +339,17 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
             keyInfoList
                 .add(KeyInfo(commitEntry.atKey, SyncDirection.localToRemote));
           } on Exception catch (e) {
+            var cause = (e is AtException) ? e.getTraceMessage() : e.toString();
             _logger.severe(
-                'exception while updating commit entry for entry:$entry ${e.toString()}');
+                'exception while updating commit entry for entry:$entry Reason: $cause');
+            return keyInfoList;
           }
         }
       } on Exception catch (e) {
+        var cause = (e is AtException) ? e.getTraceMessage() : e.toString();
         _logger.severe(
-            'exception while syncing batch: ${e.toString()} batch commit entries: $unCommittedEntryList');
+            'exception occurred while syncing batch commit entries: $unCommittedEntryList  Reason: $cause');
+        return keyInfoList;
```

Comment:
check my latest commit to trunk..we should not return keyInfoList on exception

Reply:
Removed the return statement